pilot/pkg/config/clusterregistry/clusterconfig.gotmpl

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+clusters:
```

Comment:
I'd think there's already an API in a k8s go package that would handle the creation of kubeconfig.  I don't think we want to introduce our own version here in istio.

Reply:
@tiswanso yep, there is but these template files are used here just in conversion_test.g, so  just to run unit tests and to not clutter go code with static data.